Database/Corrections/sodQuestFixes.lua

```diff
@@ -5709,6 +5709,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -439767,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90310] = {
+            [questKeys.name] = "Axe Specialization",
+            [questKeys.startedBy] = {nil,{457101}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 50,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER + classIDs.PALADIN + classIDs.SHAMAN + classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Read the book 'The Rites of Mak'Gora' to receive the rune."},
+            [questKeys.requiredSpell] = -453688,
+            [questKeys.zoneOrSort] = classIDs.HUNTER + classIDs.PALADIN + classIDs.SHAMAN + classIDs.WARRIOR,
```

Comment:
This won't work. The `zoneOrtSort` field must always be a single zone, class or profession ID. Since the object is in "Burning Steps" I would add it there as well:

```suggestion
            [questKeys.zoneOrSort] = zoneIDs.BURNING_STEPPES,
```